README.md

```diff
@@ -91,27 +94,26 @@ trust-on-first-use, servers that automatically generate self-signed certificates
 should write their generated certificate to persistent storage and reuse it on
 future runs.
 
-## Running the Examples
+</details>
+<p></p>
 
-```sh
-$ cargo run --example server ./
-$ cargo run --example client https://localhost:4433/Cargo.toml
-```
+## Contribution
 
-This launches a HTTP 0.9 server on the loopback address serving the current
-working directory, with the client fetching `./Cargo.toml`. By default, the
-server generates a self-signed certificate and stores it to disk, where the
-client will automatically find and trust it.
-
-## Development
-
-The quinn-proto test suite uses simulated IO for reproducibility and to avoid
+- All feedback welcome. Feel free to file bugs, requests for documentation and
+  any other feedback to the [issue tracker][issues].
+- The quinn-proto test suite uses simulated IO for reproducibility and to avoid
```

Comment:
I'd drop the bullet points here in favor of a paragraph break.

Reply:
@TimonPost if you process this I think this is ready to merge!

Reply:
@djc I progressed this in 1fa2a43

Reply:
On https://github.com/TimonPost/quinn/tree/timon/read-me-update I still see the bullet points in the Contribution section.

Reply:
o whoops my bad. I changed the wrong bullet points I see

Reply:
Ah yeah, please change those back. :)

Reply:
Fixed it 👍